atlp-rwanda/script-heroes-bn-backend
# #175973595 Redirect to home page after success login

What does this PR do?
User should be redirected to the landing page after login success

Description of Task to be completed?
- Save token in cookie
- Add a redirection to the landing page

How should this be manually tested?
- Clone this repo git clone https://github.com/atlp-rwanda/script-heroes-bn-backend
- Checkout the branch git checkout fix-login-redirect-to-landing-page
- Run the app npm run dev
- Log out from Facebook & Google in your browser
- Navigate to http://localhost:3000/users/auth/facebook for Facebook Login
- Or http://localhost:3000/users/auth/google for Google Login
- Insert the credentials
- Check if it redirects to the Landing page on heroku

When login success you will be redirected to the landing Page

What are the relevant pivotal tracker stories?
[#175973595](https://www.pivotaltracker.com/story/show/175973595)
